emqx/assets/dashboards/overview.json

```diff
@@ -0,0 +1 @@
itle_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"},{"formula":"query2"},{"formula":"query3"}],"queries":[{"data_source":"metrics","name":"query1","query":"sum:emqx.packets.connect.count{*}.as_rate()"},{"data_source":"metrics","name":"query2","query":"sum:emqx.packets.connack.sent.count{*}.as_rate()"},{"data_source":"metrics","name":"query3","query":"sum:emqx.packets.connack.error.count{*}.as_rate()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":6,"y":15,"width":6,"height":3}},{"id":8957776368672518,"definition":{"title":"Publish Packets","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"},{"formula":"query2"},{"formula":"query3"},{"formula":"query4"},{"formula":"query5"}],"queries":[{"data_source":"metrics","name":"query1","query":"sum:emqx.packets.publish.received.count{*}.as_rate()"},{"data_source":"metrics","name":"query2","query":"sum:emqx.packets.publish.sent.count{*}.as_rate()"},{"data_source":"metrics","name":"query3","query":"sum:emqx.packets.publish.dropped.count{*}.as_rate()"},{"data_source":"metrics","name":"query4","query":"sum:emqx.packets.publish.error.count{*}.as_rate()"},{"data_source":"metrics","name":"query5","query":"sum:emqx.packets.puback.inuse.count{*}.as_rate()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":0,"y":18,"width":6,"height":3}},{"id":6466172691344548,"definition":{"title":"Subscribe Packets","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"},{"formula":"query2"},{"formula":"query3"},{"formula":"query4"},{"formula":"query5"}],"queries":[{"data_source":"metrics","name":"query1","query":"sum:emqx.packets.subscribe.received.count{*}.as_rate()"},{"data_source":"metrics","name":"query2","query":"sum:emqx.packets.suback.sent.count{*}.as_rate()"},{"data_source":"metrics","name":"query3","query":"sum:emqx.packets.subscribe.error.count{*}.as_rate()"},{"data_source":"metrics","name":"query4","query":"sum:emqx.packets.unsubscribe.received.count{*}.as_rate()"},{"data_source":"metrics","name":"query5","query":"sum:emqx.packets.unsubscribe.error.count{*}.as_rate()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":6,"y":18,"width":6,"height":3}}],"template_variables":[{"name":"node","prefix":"endpoint","available_values":[],"default":"*"}],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed"}
```

Comment:
The description should match the overview section within the dashboard. for example: This dashboard provides an overview of your EMQX environment. It monitors node status, provides information on message status, and provides insight into various other EMQX setup observability measures.

Reply:
The title should be : EMQX - Overview

Reply:
@emarsha94 Thanks for reviewing. done!